version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.2",
+  "version": "2.5-prerelease",
```

Comment:
Note: this change also exists in #1912 - whichever lands first wins the v2.5-prelease versioning kick.